collection-input/cesm1-le-collection.yml

```diff
@@ -237,6 +246,47 @@ data_sources:
             member_id: 105
             has_ocean_bgc: True
             ctrl_branch_year: 402
+          - case: b.e11.B20TRC5CNBDRD.f09_g16.OIC.001d
+            sequence_order: 0
+            member_id: 1
+            has_ocean_bgc: False
```

Comment:
Pretty sure these should all be `has_ocean_bgc = True`

Reply:
You can tell by searching for fields like `DIC` or `FG_CO2`....if these are present, the case has BGC. I wonder, however, if we should just remove this field: if a case has a particular variable, it will be returned in a query; otherwise, it won't be.

Reply:
> I wonder, however, if we should just remove this field: if a case has a particular variable, it will be returned in a query; otherwise, it won't be.

If the `has_ocean_bgc` field doesn't add any additional information, I favor removing it.